actioncable/lib/action_cable.rb

```diff
@@ -35,7 +35,8 @@ module ActionCable
       confirmation: 'confirm_subscription'.freeze,
       rejection: 'reject_subscription'.freeze
     },
-    default_mount_path: '/cable'.freeze
+    default_mount_path: '/cable'.freeze,
+    protocols: ["actioncable-v1-json".freeze, "actioncable-unsupported".freeze].freeze
```

Comment:
Apologies for asking again since I think you explained this already, but it's not clear to me what the purpose of actioncable-unsupported protocol is. Perhaps it could be defined as a separate constant that gets included in this array so it can be documented separately.

Additionally, now that we have a v1 protocol in place, what kinds of changes warrant bumping the version? (I know the answer, but think it's worth documenting).